cli/index.js

```diff
@@ -0,0 +1,204 @@
+// This is a CLI tool, using console is OK
+/* eslint no-console: 0 */
+const {spawn, exec} = require('child_process');
+const {isAbsolute, resolve} = require('path');
+const {existsSync} = require('fs');
+const pify = require('pify');
+const args = require('args');
+const chalk = require('chalk');
+const opn = require('opn');
+// const columnify = require('columnify');
+// const got = require('got');
+// const ora = require('ora');
+const api = require('./api');
+
+let commandPromise;
+
+const checkConfig = () => {
+  if (api.exists()) {
+    return true;
+  }
+  let msg = chalk.red(`Error! Config file not found: ${api.configPath}\n`);
+  msg += 'Please launch Hyper and retry.';
```

Comment:
```js
const message = `
${chalk.red(`Error! Config file not found: ${api.configPath}`)}
Please launch Hyper and retry.
`
```

Reply:
Should we add an issue to make the cli ask to create the file if it does not exists?

Reply:
Yes. 
Config creation should remain an `Hyper` task. And yes, we should improve this automatic creation. But this is not a blocker. Most of case, config file will be present.

Reply:
👌